examples/dcmip_3-1_linear_advection_triangle_bdfm1.py

```diff
@@ -40,12 +40,14 @@
 timestepping = TimesteppingParameters(dt=10.0)
 output = OutputParameters(Verbose=True, dumpfreq=1, dirname='dcmip')
 parameters = CompressibleParameters(k=k, Omega=Omega)
+diagnostics = Diagnostics(fieldlist)
```

Comment:
I think that these fields will be registered twice (once in initialisation here) and then again in State.dump?